src/osx/carbon/graphics.cpp

```diff
@@ -2333,10 +2322,9 @@ void wxMacCoreGraphicsContext::GetTextExtent( const wxString &str, wxDouble *wid
     {
         if ( width )
             *width = w;
-        if ( height )
-            *height = a+d+l;
     }
-
+    if ( height )
```

Comment:
Not sure why has this been moved? It seems logical to return height of 0 for an empty string and I suspect that the other ports (still) do the same. If we do need to change it, it would be better to do it in a separate commit and, ideally, with a unit test checking for this, so that we could ensure the behaviour is consistent across all ports.